### PR TITLE
fix: answer of HTML quiz Q106 is incorrect

### PR DESCRIPTION
**Pull request by oorkan**
## PR Checklist

This PR is ready for review and meets the requirements set out
in [Suggestion how to contribute](CONTRIBUTING.md)

### DOD

- [ ] I have added new quiz{'s}
- [ ] I have added new reference link{'s}
- [x] I have made small correction/improvements

### Changes / Instructions

The answer to HTML quiz question 106 is incorrect.

[According to MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/th):

```
The <th> HTML element defines a cell as the header of a group of table cells. The exact nature of this group is defined by the scope and headers attributes.
```

and 

image

Plus, when we compare the options **A** and **B**, here's what we get in the browser:

**option A** (❌ **incorrect**)

image

**options B** (✅ **correct**)

image

Summing up, the correct answer is option **B**:

```html
<table>
  <tr>
    <th scope="row">Header</th>
    <td>10</td>
    <td>18</td>
  </tr>
</table>
```

Thank you!

### CODE REVIEW

**Review by Ebazhanov — APPROVED** [review]
Great job, thank you @oorkan! 👍
